Makefile

```diff
@@ -1,8 +1,12 @@
-.PHONY: test bins clean cover cover_ci unit-test
```

Comment:
Approximately 80% of our targets are `.PHONY`. So I should list all of them here or just remove this completely because we don't have files/dirs with these names.

Reply:
Ok! No opinion on this.

Reply:
Ok! No opinion on this.

Reply:
Ok! No opinion on this.